src/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Can we pluralize the filename to be inline with the naming convention.
`src/Http/Controllers/CategoryController.php` -> `src/Http/Controllers/CategoriesController.php`